### DISCUSSION TITLE
setValue on field array Bug? w/ 6.9.4

### DISCUSSION THREAD

**Discussion by fadi-george** [react-hook-form/react-hook-form#3208] [Q&A] [answered]
Hello,

I just ran into this after upgrading to 6.9.4 from 6.9.2.

Steps:
- click append twice
- click the edit button in the first row
- name of the first item should get updated to `Updated Q1`

v6.9.2:
https://codesandbox.io/s/react-hook-form-692-setvalue-o7chd

v6.9.4
https://codesandbox.io/s/react-hook-form-694-setvalue-forked-7wjqz?file=/src/index.js

**Comment by bluebill1049** [comment, marked as answer]
Thanks @fadi-george would you like to convert this into an issue? i will take a deeper look later on, in case i forgot.

**Reply by fadi-george** [reply]
Done!

**Reply by bluebill1049** [reply]
ta
